internal/controller/siteconfig_builder.go

```diff
@@ -107,48 +107,48 @@ func (scbuilder *SiteConfigBuilder) renderTemplates(ctx context.Context, c clien
 		}
 
 		// process Template ConfigMap
-		for kind, template := range templatesConfigMap.Data {
-			if suppressRenderingManifest(kind, suppressedManifests) {
-				scbuilder.Log.Info(fmt.Sprintf("renderTemplates: skipping rendering template manifest %s for SiteConfig %s", kind, siteConfig.Name))
-				continue
-			}
+		for templateKey, template := range templatesConfigMap.Data {
 
 			siteData, err := buildSiteData(siteConfig, node)
 			if err != nil {
 				scbuilder.Log.Error(err, fmt.Sprintf("renderTemplates: failed to build SiteConfig data for SiteConfig %s", siteConfig.Name))
 				return nil, err
 			}
 
-			manifest, err := scbuilder.render(kind, template, siteData)
+			manifest, err := scbuilder.render(templateKey, template, siteData)
 			if err != nil {
 				scbuilder.Log.Error(err, fmt.Sprintf("renderTemplates: failed to render templateRef %s for SiteConfig %s", templateRef.Name, siteConfig.Name))
 				return manifests, err
 			}
 
 			if manifest != nil {
```

Comment:
Also here it might simplify things to use a guard clause now that I'm thinking about it.

Reply:
The template kind is only parsed in the `scbuilder.render` function.

Reply:
Sure, but this line could be:
```
if manifest == nil {
  continue
}
```

Then the rest of the loop wouldn't need to be indented.

Reply:
Changed :)